.github/pull_request_template.md

```diff
@@ -1,5 +1,16 @@
 🎉 Thanks for submitting a pull request! 🎉
 
+#### Summary
+
+Fixes #<replace_with_issue_number>
+
+<!--
+Explain the **motivation** for making this change.
+What existing problem does the pull request solve and how?
+-->
+
+---
+
 For us to review and ship your PR efficiently, please perform the following steps:
 
 - [ ] Open a [bug/issue](https://github.com/netlify/build-image/issues/new/choose) before writing your code 🧑‍💻. This ensures
```

Comment:
Could we remove the line wrapping?
I am not sure it makes sense considering the default font is not fixed width (although this can now be customized) and also in the context of a web page.